analyzer/consensus/consensus_test.go

```diff
@@ -97,5 +97,5 @@ func TestMalformedTxMethod(t *testing.T) {
 	parsed, err := unpackTxBody(&tx)
 	require.Nil(t, parsed)
 	require.NotNil(t, err)
-	require.Equal(t, "unknown tx method: not_a_valid_method", err.Error())
+	require.Equal(t, "unable to cbor-decode consensus tx body: %!w(<nil>), method: not_a_valid_method, body: deadbeef", err.Error())
```

Comment:
the "<nil>" suggests we should initialize `error` in `consensus.go:382` to `fmt.Errorf("unknown tx method")`?